rust/src/scannerctl/interpret/mod.rs

```
@@ -116,11 +119,25 @@ async fn run_on_storage<S: ContextStorage, L: Loader>(
     storage: S,
     loader: L,
     target: Target,
+    kb: Vec<String>,
     ports: Ports,
     script: &Path,
 ) -> Result<(), CliErrorKind> {
     let scan_id = ScanID(format!("scannerctl-{}", script.to_string_lossy()));
     let filename = script;
+    let kbctx = (
+        scan_id.clone(),
+        scannerlib::storage::Target(target.ip_addr().to_string()),
+    );
+    kb.iter().for_each(|s| {
+        if let Some((k, v)) = s.split_once("=") {
+            let storage_ctx = KbContextKey(kbctx.clone(), k.into());
+            let _ = storage
+                .dispatch(storage_ctx, v.into())
+                .map_err(CliErrorKind::StorageError);
```

Should we not handle the error here? Since this is scannerctl, I think even an `unwrap` is fine, but silently ignoring it feels wrong.
If you want to return the result, changing the `for_each` to a for loop will also make it easier